test/bin/scenario.sh

```diff
@@ -147,12 +151,8 @@ launch_vm() {
          --wait
 
     # Wait for an IP to be assigned
-    ip=$(get_vm_ip "${full_vmname}")
-    while [ -z "${ip}" ]; do
-        echo "Waiting for VM ${full_vmname} to have an IP"
-        sleep 30
-        ip=$(get_vm_ip "${full_vmname}")
-    done
+    echo "Waiting for VM ${full_vmname} to have an IP"
+    ip=$(timeout 300 bash -c "get_vm_ip ${full_vmname}")
```

Comment:
We're using the same timeout everywhere else.

```suggestion
    ip=$(timeout "${VM_BOOT_TIMEOUT}" bash -c "get_vm_ip ${full_vmname}")
```